sumo_deployments/helm/sumo-opentelemetry-demo/templates/frontend.yaml

```diff
@@ -0,0 +1,112 @@
+    app.kubernetes.io/part-of: opentelemetry-demo
+    app.kubernetes.io/managed-by: Helm
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: '{{ include "otel-demo.name" . }}'
+      app.kubernetes.io/instance: '{{ include "otel-demo.name" . }}'
+      app.kubernetes.io/component: frontend
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: '{{ include "otel-demo.name" . }}'
+        app.kubernetes.io/instance: '{{ include "otel-demo.name" . }}'
+        app.kubernetes.io/component: frontend
+    spec:
+      containers:
+        - name: frontend
+          image: '{{ printf "%s:%s" .Values.sumologic.deployments.frontend.image.repository .Values.sumologic.deployments.frontend.image.tag }}'
+          imagePullPolicy: IfNotPresent
+          ports:
+            - containerPort: 8080
+              name: service
+          env:
+            - name: OTEL_SERVICE_NAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: metadata.labels['app.kubernetes.io/component']
+            - name: OTEL_K8S_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: metadata.namespace
+            - name: OTEL_K8S_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
+            - name: OTEL_K8S_POD_NAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: metadata.name
+            - name: FRONTEND_ADDR
+              value: :8080
+            - name: AD_SERVICE_ADDR
+              value: '{{ include "otel-demo.name" . }}-adservice:8080'
+            - name: CART_SERVICE_ADDR
+              value: '{{ include "otel-demo.name" . }}-cartservice:8080'
+            - name: CHECKOUT_SERVICE_ADDR
+              value: '{{ include "otel-demo.name" . }}-checkoutservice:8080'
+            - name: CURRENCY_SERVICE_ADDR
+              value: '{{ include "otel-demo.name" . }}-currencyservice:8080'
+            - name: PRODUCT_CATALOG_SERVICE_ADDR
+              value: '{{ include "otel-demo.name" . }}-productcatalogservice:8080'
+            - name: RECOMMENDATION_SERVICE_ADDR
+              value: '{{ include "otel-demo.name" . }}-recommendationservice:8080'
+            - name: SHIPPING_SERVICE_ADDR
+              value: '{{ include "otel-demo.name" . }}-shippingservice:8080'
+            - name: OTEL_EXPORTER_OTLP_ENDPOINT
+              value: http://{{ include "sumo-opentelemetry-demo-otelagent-svc-name" . }}:4317
+            - name: FRONTEND_PORT
+              value: "8080"
+            - name: SUMO_LOGIC_RUM_ENABLED
+              value: "true"
+            - name: PUBLIC_SUMO_LOGIC_RUM_SERVICE_NAME
+              value: frontend-rum
+            - name: PUBLIC_SUMO_LOGIC_RUM_APPLICATION_NAME
+              value: opentelemetry-demo
```

Comment:
Would be nicer if these were all includes from _helpers.tpl, especially the Service names.

Reply:
I agree but for now the goal is to just run is asap and provide demo data to Sumo to present our capabilities :) I will prepare a TODO task for that in future.